packager/media/codecs/es_descriptor.h

```diff
@@ -175,7 +175,7 @@ class ESDescriptor : public BaseDescriptor {
   ESDescriptor() : BaseDescriptor(DescriptorTag::kES) {}
 
   uint16_t esid() const { return esid_; }
-  void set_esid(uint16_t esid) { esid_ = esid; }
+  void set_esid(uint16_t esid) { esid_ = 0; }
```

Comment:
This is not the correct way to address it.

Since we are not expecting the users to modify it, we should remove this function. We should also update ESDescriptor::WriteInternal to write "0" instead of writing |esid_|.